lib/components/Snackbar/snackbar.scss

```diff
@@ -0,0 +1,97 @@
+@import "@quill/spacing.scss";
+@import "@quill/border.scss";
+@import "@quill/breakpoints.scss";
+@import "@quill/motion.scss";
+@import "@quill/elevation.scss";
+
+.snackbar {
+    display: flex;
+    width: 340px;
```

Comment:
can we remove width for mobile devices. For smaller mobiles, width 340px will go out of the container.

Reply:
should I? i'm trying to follow figma design

Reply:
yeah, it's better to do something like this:
```
left: 16px;
width: calc(100vw - 32px);
```
and i guess all `px` values in this file should be replaced with `var()`'s :)

Reply:
why's it left: 16px? It'll be centered by the wrapper.